receiver/kubeletstatsreceiver/kubelet/mem.go

```diff
@@ -38,3 +41,15 @@ func memUsageMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
 func memRssMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
 	return intGauge(prefix+"memory.rss", "By", s.RSSBytes)
 }
+
+func memWorkingSetMetric(prefix string, s *stats.MemoryStats) *metricspb.Metric {
+	return intGauge(prefix+"memory.workingset", "By", s.WorkingSetBytes)
```

Comment:
And for consistency with others we probably need `memory.working_set` here

Reply:
Fixed.